### ISSUE TITLE
Missing values when language not identified

### ISSUE THREAD

**Issue by kand617** [rexxars/react-lowlight#1]
I am seeing a strange behavior in some cases for code elements....
It looks like it renders an empty tag when it is unable to identify the language. 
For example: 

![image](https://cloud.githubusercontent.com/assets/5092518/23586795/e400e74c-0201-11e7-8cc8-a2275510b97b.png)

I have registered the following languages
```javascript
Lowlight.registerLanguage('js', js);
Lowlight.registerLanguage('php', php);
Lowlight.registerLanguage('python', python);
Lowlight.registerLanguage('ruby', ruby);
Lowlight.registerLanguage('go', go);
Lowlight.registerLanguage('cs', cs);
Lowlight.registerLanguage('java', java);
Lowlight.registerLanguage('php', php);
Lowlight.registerLanguage('bash', Bash);
Lowlight.registerLanguage('json', JSONLang);
Lowlight.registerLanguage('objectivec', ObjC);
```

On another note, I am really surprised that lowlight is not able to infer the language. 
Highlight js can...

Thats odd. Just tested against https://github.com/conorhastings/react-syntax-highlighter 
Renders fine on that plugin...

**Comment by rexxars** [comment]
Hrm. First of all, the fact that *nothing* is rendered is a bug - obviously it should fall back to just printing the text as-is.

Secondly, it seems to have to do with not being able to detect the language based on the limited snippet. I registered all the available languages from highlight js, which results in the snippet being detected as [ebnf](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form). When I removed ebnf as a registrered language, it fell back to (in order): abnf, axapta, fix, mipsasm, nginx, nix, protobuf. When I removed all of those, it didn't manage to guess a language anymore.

Publishing a fix in a second.
